src/components/accessories/feedback/type.ts

```diff
@@ -0,0 +1,7 @@
+export interface IFeedbackProps {
+  enabled: boolean;
+  url: string;
+  email: string;
+  version: string;
+  priority: string;
+}
```

Comment:
The `IFeedbackProps` should be moved into the directory `/customization/feedback` in a ts file; something like `types.ts`. Then, from the index file, you can export all the types from this `types.ts` file.